app/src/main/java/net/squanchy/speaker/SpeakerDetailsActivity.kt

```diff
@@ -0,0 +1,137 @@
+package net.squanchy.speaker
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.support.v7.widget.Toolbar
+import android.view.Menu
+import android.view.MenuItem
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import kotlinx.android.synthetic.main.activity_speaker_details.*
+import net.squanchy.R
+import net.squanchy.navigation.Navigator
+import net.squanchy.speaker.domain.view.Speaker
+import net.squanchy.support.config.DialogLayoutParameters
+import net.squanchy.support.lang.Optional
+
+class SpeakerDetailsActivity : AppCompatActivity() {
+
+    private var subscriptions = CompositeDisposable()
```

Comment:
I think this could be `val` and we call `clear()` instead of `dispose()` without creating a new instance down below, in `onNewIntent()` and `observeSpeakerFrom()`